client/webpack.config.js

```diff
@@ -1,3 +1,6 @@
+const webpack = require('webpack');
+const minimise = process.argv.indexOf('--dev') === -1;
```

Comment:
Why only on dev - it can be problematic to have different files for different envs.

If the reason is that source maps don't work with this, it's also problematic as we rely on those for sentry to be able to debug real-time errors efficiently.